Automattic/wp-calypso
# User invite: Make invite passwordless

Fixes https://github.com/Automattic/wp-calypso/issues/86757

## Proposed Changes

* Makes the accept invite form passwordless when receiving an invitation from wp-admin
* There is also passwordless signup mostly used on onboarding flows, it just creates the user. This PR is not using that, but the regular accept invite form. We need to create the user, accept the invitation and redirect to the proper page.


## Testing Instructions



* Go to `https://wordpress.com/people/team/[site-slug]`
* Invite a user who has no WordPress account (I am using a test email creator, ping me if you need it)
* Open the invitation in incognito mode
* Replace wordpress.com with calypso:3000/
* You should see a passwordless form
* Accept the invitation, should bring you to my home on Wp-admin
* Checking on your atomic, this user should be in the list

Please reach me out if it is not clear how to test the steps, it is a bit complicated indeed.
